pkgs/development/compilers/llvm/16/libcxx/default.nix

```
@@ -62,7 +60,10 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ cmake ninja python3 ]
     ++ lib.optional stdenv.isDarwin fixDarwinDylibNames;
 
-  buildInputs = lib.optionals (!headersOnly) [ cxxabi ];
+  buildInputs = (
+    lib.optional (!headersOnly) cxxabi ++
```

Unless the optional is only ever going to be one thing, it seems better to leave it as a list. This also matches the the style used elsewhere in the LLVM derivations.

```suggestion
  buildInputs = lib.optionals (!headersOnly) [ cxxabi ]
```